lxd/containers.go

```diff
@@ -757,27 +757,20 @@ func (d *lxdContainer) applyConfig(config map[string]string) error {
 	}
 
 	if lxcConfig, ok := config["raw.lxc"]; ok {
-		for _, line := range strings.Split(lxcConfig, "\n") {
-			contents := strings.SplitN(line, "=", 2)
-			key := contents[0]
-			value := ""
-			if len(contents) > 1 {
-				value = contents[1]
-			}
-
-			key = strings.Trim(key, " ")
-			value = strings.Trim(value, " ")
-
-			// empty lines
-			if key == "" {
-				continue
-			}
+		f, err := ioutil.TempFile("", "lxd_config")
```

Comment:
On Wed, Mar 11, 2015 at 10:16:45AM -0700, St�phane Graber wrote:
> > -		for _, line := range strings.Split(lxcConfig, "\n") {
> > -			contents := strings.SplitN(line, "=", 2)
> > -			key := contents[0]
> > -			value := ""
> > -			if len(contents) > 1 {
> > -				value = contents[1]
> > -			}
> > -
> > -			key = strings.Trim(key, " ")
> > -			value = strings.Trim(value, " ")
> > -
> > -			// empty lines
> > -			if key == "" {
> > -				continue
> > -			}
> > +		f, err := ioutil.TempFile("", "lxd_config")
> 
> Does TempFile add a separate (_ or something else) for us there?
> 
> A grep through our current code shows that we've been using a trailing underscore in our prefixes so far.

It doesn't, it just adds a number at the end. We're using trailing
underscores in some places and not others.

> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/lxc/lxd/pull/365/files#r26233286